app/flatpak-builtins-ls-remote.c

```diff
@@ -283,7 +285,8 @@ flatpak_builtin_ls_remote (int argc, char **argv, GCancellable *cancellable, GEr
               strcmp (arches[0], parts[2]) != 0)
             {
               g_autofree char *alt_arch_ref = g_strconcat (parts[0], "/", parts[1], "/", arches[0], "/", parts[3], NULL);
-              if (g_hash_table_lookup (refs, alt_arch_ref))
+              g_autoptr(FlatpakCollectionRef) alt_arch_coll_ref = flatpak_collection_ref_new (NULL, alt_arch_ref);
```

Comment:
Surely this should use the collection id from coll_ref? We're looking for another ref that is the same as coll_ref, but with a different arch only.

Reply:
Yeah, pushed a fixup